tests/utils/test_http_client.py

```diff
@@ -23,6 +23,5 @@ def test_header(self):
         cli = LaunchableClient("/test", test_runner="dummy")
         self.assertEqual(cli._headers(False), {
             'Content-Type': 'application/json',
-            "User-Agent": "Launchable/{} (Python {}, {})".format(__version__, platform.python_version(), platform.platform()),
-            "launchable-test-runner": "dummy"
+            "User-Agent": "Launchable/{} (Python {}, {}) (TestRunner: {})".format(__version__, platform.python_version(), platform.platform(), "dummy"),
```

Comment:
Sorry to nitpicking but `(Python {}, {})` does not contain any `:` but `(TestRunner: {})` does. I think either you should remove `:` from test runner or add `:` to the Python output.

Reply:
OK, I remove `:`